releases/release-1.31/release-notes/maps/pr-124465-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 124465
+releasenote:
+  text: 'kubeadm: switch to using the new etcd endpoints introduced in 3.5.11 - /livez
+    (for liveness probe) and /readyz (for readyness and startup probe). With this
+    change it is no longer possible to deploy a custom etcd version older than 3.5.11
+    with kubeadm 1.31. If so, please upgrade.'
```

```suggestion
    with kubeadm 1.31. If so, please upgrade etcd to a supported version.'
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653314905